packages/dev/core/src/Shaders/picking.vertex.fx

```diff
@@ -0,0 +1,31 @@
+﻿// Attributes
+attribute vec3 position;
+#if defined(INSTANCES)
+attribute vec4 instanceMeshID;
+#endif
+
+#include<bonesDeclaration>
+#include<bakedVertexAnimationDeclaration>
+
+// Uniforms
+
+#include<instancesDeclaration>
+uniform mat4 viewProjection;
+
+// Output
+#if defined(INSTANCES)
+varying vec4 vMeshID;
+#endif
+
+void main(void) {

```

Comment:
Support for morph targets is missing:
* before `void main`
```glsl
#include<morphTargetsVertexGlobalDeclaration>
#include<morphTargetsVertexDeclaration>[0..maxSimultaneousMorphTargets]
```
* before `#include<instancesVertex>`:
```glsl
#include<morphTargetsVertexGlobal>
#include<morphTargetsVertex>[0..maxSimultaneousMorphTargets]
```

Reply:
oupsie :)

Reply:
oh we do not have it in color shader either...